### PR TITLE
Docs: Updating the macOS guide for 'New Terminal at Folder'

### PR DESCRIPTION
**Pull request by Bicoloursnake**
# Docs: Enhancing the macOS guide

## What is this fixing or adding?

Adding a small sentence to the mac guide that provides an alternative to navigating directories/folders in the terminal in an attempt to to make it more accessible.

## How was this tested?

I previewed the new markdown.

### CODE REVIEW

File: worlds/generic/docs/mac_en.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Here is a list of software to install and source code to download.
 3. Open terminal and navigate to your Archipelago directory.
 ## Setting up a Virtual Environment
 It is generally recommended that you use a virtual environment to run python based software to avoid contamination that can break some software. If Archipelago is the only piece of software you use that runs from python source code however, it is not necessary to use a virtual environment. 
-1. Open terminal and navigate to the Archipelago directory.
+1. Open terminal and navigate to the Archipelago directory. (Alternatively, right click on the Archipelago folder in Finder and select 'New Terminal at Folder')
```
Comment (black-sliver) [inline worlds/generic/docs/mac_en.md:18]:
```suggestion
1. Open terminal and navigate to the Archipelago directory. Alternatively, right click on the Archipelago folder in Finder and select 'New Terminal at Folder'.
```

**Review by black-sliver — APPROVED** [review]
I think the punctuation looks better like this?

**Review by black-sliver — CHANGES_REQUESTED** [review]
Oops. hit the wrong button. See comment above

**Comment by Bicoloursnake** [comment]
Agreed on the punctuation change. Thanks!

**Review by black-sliver — APPROVED** [review]

**Pull request merged**